website/docs/r/hdinsight_hadoop_cluster.html.markdown

```diff
@@ -247,6 +249,54 @@ A `install_script_action` block supports the following:
 
 * `uri` - (Required) The URI pointing to the script to run during the installation of the edge node. Changing this forces a new resource to be created.
 
+--- 
+
+A `metastores` block supports the following:
+
+* `hive` - (Required) A `hive` block as defined below.
+
+* `oozie` - (Required) A `oozie` block as defined below.
+
+* `ambari` - (Required) A `amabari` block as defined below.
```

Comment:
The PR description says "the metastores are optional and completely independent." However, the documentation seems to list all three metastore blocks as _Required_ if the `metastores` block is defined.